tests/trainer/test_optimizers.py

```diff
@@ -126,55 +126,60 @@ def test_multi_optimizer_with_scheduling_stepping(tmpdir):
         'lr for optimizer 2 not adjusted correctly'
 
 
-def test_reduce_lr_on_plateau_scheduling_missing_monitor(tmpdir):
-
-    hparams = EvalModelTemplate.get_default_hparams()
-    model = EvalModelTemplate(**hparams)
-    model.configure_optimizers = model.configure_optimizers__reduce_lr_on_plateau
-
-    # fit model
-    trainer = Trainer(
-        default_root_dir=tmpdir,
-        max_epochs=1,
-        limit_val_batches=0.1,
-        limit_train_batches=0.2,
-    )
-
-    m = '.*ReduceLROnPlateau requires returning a dict from configure_optimizers.*'
-    with pytest.raises(MisconfigurationException, match=m):
+def test_reducelronplateau_with_no_monitor_raises(tmpdir):
+    """
+    Test exception when a ReduceLROnPlateau is used with no monitor
+    """
+    model = EvalModelTemplate()
+    optimizer = torch.optim.Adam(model.parameters())
+    model.configure_optimizers = lambda: ([optimizer], [torch.optim.lr_scheduler.ReduceLROnPlateau(optimizer)])
```

Comment:
```suggestion
    model.configure_optimizers = model.configure_optimizers__reduce_lr_on_plateau
```

Reply:
I actually removed it from the pool because it was not being used anywhere else

Reply:
it might be used sometime later since it's a pretty common one.

Reply:
Well, the way it was implemented does not work (notice this is a `_raises` test), so I don't think it will be used anywhere else

Reply:
ok ok yeah. the `monitor` is required.. haha didn't see that. all good :)